metrics/auto_increment/auto_increment.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2024 Block, Inc.
```

Comment:
s/2024/2025/. You can probably update this in all files.

Reply:
I'll make a todo for this in a separate PR.